bosh/init.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+env=${DEPLOY_ENV:-$1}
+DEPLOYER_CONCOURSE=${DEPLOYER_CONCOURSE:-$2}
+SCRIPT_DIR=$(cd "$(dirname $0)" && pwd)
+BOSH_PORT=${BOSH_PORT:-"25555"}
+
+[ -z "${env}" ]                && echo "Must provide DEPLOY_ENV as \$1 or env. var"                        && exit 101
+[ -z "${DEPLOYER_CONCOURSE}" ] && echo "Must provide DEPLOYER_CONCOURSE ip or hostname as \$2 or env. var" && exit 102
+
+pushd ${SCRIPT_DIR}
+
+  # TODO: check if you really want/need to re-create the session
+  #rm session/* && rmdir session
+  [ ! -d session ] && mkdir session
+  pushd session
+    # Get the key
+    ../../concourse/scripts/s3get.sh ${env}-state id_rsa > /dev/null
+    chmod 600 id_rsa
+
+    # Create ssh tunnel to concourse if not present yet
+    if [[ ! $(netstat -an | grep 127.0.0.1.25555) ]]; then
```

Comment:
In my experience this implementation can easily end with ssh running but not working, and I am not sure if greping the `netstat` output  will work fine on macosx and on systems with IPv6. 

I did not have bath experience with control sockets, unless you delete the socket. In that case, it would be the same than not using the socket at all. The control socket gives you a communication channel to a running SSH.

Anyway, it is worth it to `echo` some messages saying that the tunnel was created (or not) and how to kill it.